tools/zstpak/lib/zstpak.go

```diff
@@ -183,7 +183,7 @@ func Open(path string) (*Reader, error) {
 
 	hdr, err := readFooter(file, uint64(fileInfo.Size()))
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("%s: %v,", path, err)
```

Comment:
The above error messages do not add this either. If looking at `os.Open` and other similar places, the they don't do this either. Typically adding the filename is the responsibility of the caller. Or at least should be made consistent. But since this method is `Open` I'd make the error strings similar to `os.Open` and have caller append the filename if needed.